### ISSUE TITLE
Add-EflowNetwork get error in AzureIoTEdge_CR_X64_1.2.10.09062.msi

### ISSUE THREAD

**Issue by AiHaibara** [Azure/iotedge-eflow#176]
**Describe the bug**
AzureIoTEdge_CR_X64_1.2.10.09062.msi downloaded from https://aka.ms/AzEFLOWMSI-CR-X64 in doc.
and Add-EflowNetwork get error as follow:

**To Reproduce**
```
$msiPath = $([io.Path]::Combine($env:TEMP, 'AzureIoTEdge.msi'))
$ProgressPreference = 'SilentlyContinue'
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
Invoke-WebRequest "https://aka.ms/AzEFLOWMSI-CR-X64" -OutFile $msiPath
Start-Process -Wait msiexec -ArgumentList "/i","$([io.Path]::Combine($env:TEMP, 'AzureIoTEdge.msi'))","/qn"
```
Add VM Switch from hyper-v Manager Tool
`Get-VMSwtich` get the name list
`Add-EflowNetwork` using the name from list.

**Expected behavior**
Get-VmSwitch

Name           SwitchType
VmNAT          Internal

Add-EflowNetwork

cmdlet Add-EflowNetwork at command pipeline position 1
Supply values for the following parameters:
vswitchName: VmNAT
Get-EflowVmSwitch: Cannot validate argument on parameter 'switchType'. The argument "" does not belong to the set "External,Internal"
specified by the ValidateSet attribute. Supply an argument that is in the set and then try the command again.

Exception: The virtual switch 'VmNAT' of type 'Internal' is not found
 
**Additional context**
```
function Add-EflowNetwork
{
    param (
        [Parameter(mandatory = $true)]
        [ValidateNotNullOrEmpty()]
        [String] $vswitchName,
        [ValidateSet("External", "Internal")]
        [String] $vswitchType = $script:VmmsSwitchType_Internal
    )

    Load-Configuration

    $existing = Get-EflowVmSwitch -switchType $switchType -switchName $switchName
    
    xxx
}
```

this is the source code at AzureEFLOW.psm1
when I change the name of **switchType** to **vswitchType** and **switchName** to **vswitchName**
and open a new terminal.
it's ok for add this.

**Comment by fcabrera23** [comment]
Hi @AiHaibara,

Thanks for finding this issue. Feel free to create the PR with the fix, I'll go ahead and review it. 

Thanks,
Francisco

**Comment by AiHaibara** [comment]
> 
Hi @fcabrera23 ,

I can only find local path in
C:\Program Files\WindowsPowerShell\Modules\AzureEFLOW\AzureEFLOW.psm1
but I can't find the corresponding module file path on github.
I don't know how to submit the pull request for this issue.

Thanks,
Alan

**Comment by fcabrera23** [comment]
HI @AiHaibara,

Sorry for the inconvenience. You're probably using an old version of EFLOW. That issue was fixed a long time ago - Can you please uninstall EFLOW and reinstall it using the latest [MSI](https://aka.ms/azeflowmsi). 

Thanks,
Francisco

**Comment by embalu** [comment]
Hi I have the same problem but in my case I'm using IoT Edge and when I execute the command `Add-EflowNetwork` 

Output:
```
Get-EflowVmSwitch: Cannot validate argument on parameter 'switchType'. The argument "" does not belong to the set "External,Internal" specified by the ValidateSet attribute. Supply an argument that is in the set and then try the command again.

C:\Program Files\WindowsPowerShell\Modules\AzureEFLOW\AzureEFLOW.psm1: 7245 Carácter: 47
+     $existing = Get-EflowVmSwitch -switchType $switchType -switchName ...
+                                               ~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-EflowVmSwitch], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Get-EflowVmSwitch
```
![bugIotedge](https://user-images.githubusercontent.com/53449904/189434457-d4cb9789-9cc6-4449-bdb7-c86dcfc4528c.png)

How can I update or fix this error?

**Comment by embalu** [comment]
Ok I fixed it. 

1. I need to open this file -> C:\Program Files\WindowsPowerShell\Modules\AzureEFLOW\AzureEFLOW.psm1 with VisualCode
2. I fix the line 7245 with this code...
```
$existing = Get-EflowVmSwitch -switchType $vswitchType -switchName $vswitchName
```
![image](https://user-images.githubusercontent.com/53449904/189458003-9fa429ee-fce5-4951-9fdd-7441618ea3c6.png)


4. Restart the computer
5. Ready ✅

**Comment by fcabrera23** [comment]
Hi @embalu,

Thanks for your workaround. Just as a side note, there's no need to restart the computer. You can just reimport the EFLOW PSM module using `Import-Module AzureEFLOW -Force`, and changes should be loaded.

Thanks,
Francisco

**Comment by fcabrera23** [comment]
Hi @AiHaibara,

This issue was fixed in the latest [EFLOW 1.4LTS](https://aka.ms/AzEFLOW-Update-1_4_1_13112) version. Please use this version to get the latest feature updates and CVEs fixes.

Thanks,
Francisco
